CONTRIBUTING.md

```diff
@@ -72,7 +72,7 @@ We're creating the new branch with all the changes from `new-analysis` unstaged.
 
 ```sh
 git checkout -b new-analysis-first new-analysis
-git reset master
+git reset dev
```

Comment:
I think we want each staggered PR to have modifications in its prior staggered PRs, e.g. staggered PR part 3 should contain modifications in staggered PR part 1 and 2. Otherwise, a later part staggered PR may not be able to merge into the dev branch.

Ideally, PRs are evaluated and merged sequentially, i.e. PR 3 is not reviewed if PR 1 and 2 are not merged, as one major purpose of having staggered PRs is to isolate reviews for efficiency. This way, merging modifications may not be too complicated.

Practically, I think the reviewers and developers may need to work together to complete the review of multiple staggered PRs, so merging modifications may be carefully evaluated on a case-by-cases basis.

Reply:
I can imagine there are cases PR 3 requires changes in PR 1 and PR 2, but I think it is more efficient to handle it in PR 3 rather than leaving PR 1 and 2 open for changes. Any thoughts? @runjin326

Reply:
Gotcha! Thanks for the clarification. It makes sense!

Reply:
Oh yes didn't see your other comment - yes I think in those cases when we realize in PR3 that we need to change PR1 and PR2, we can handle that in PR3 - instead of changing from PR1 and then carry all the way over to PR3 again (while leaving PR1 and PR2 open)